lte/protos/session_manager.proto

```diff
@@ -966,10 +1021,15 @@ service SetInterfaceForUserPlane {
     rpc SetUPFSessionsConfig(UPFSessionConfigState) returns (magma.orc8r.Void) {}
 }
 
-// RPC service and method
+//SET API for 3 procedures request from AMF to SMF
+service AmfPduSessionSmContext {
+  rpc SetAmfSessionContext(SetSMSessionContext) returns (SmContextVoid) {}
+  rpc SetSmfNotification(SetSmNotificationContext) returns (SmContextVoid) {}    //AMF to SMF
+}
+// SET API RPC service and method request from SMF to AMF
 service SmfPduSessionSmContext {
  rpc SetAmfNotification(SetSmNotificationContext) returns (SmContextVoid){}    //SMF to AMF
- rpc SetSmfSessionContext(SetSMSessionContextAccess) returns (SmContextVoid);
+ rpc SetSmfSessionContext(SetSMSessionContextAccess) returns (SmContextVoid) {}
```

Comment:
why do we have this in session_manager.proto ? This is not a service provided by SMF, but is a service provided by AMF.

Reply:
Apart from AMF-SMF grpc handlers,   SMF-AMF handlers also kept ins session_manager .proto, I have just added another one. Earlier AMF team has decided to kept here,  for now  I am keeping it here.